.github/workflows/release-connect-npm-init.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           git config --global user.name "trezor-ci"
           git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
           gh config set prompt disabled
+          gh api /user --jq .login
```

Comment:
I think this should log the user that is being used here.

Reply:
Yes, but for the GH tool it's used correctly. It seems like it's not used correctly for the running script. 

the user that created the pull request was in fact trezor-ci so maybe my assumption was bad about the wrong credentials.

Reply:
I see.
Is it possible that now some sort of environment is required? 
I am trying to think in terms of auth, what have changed, since this workflow with almost same script has been working for long time and now it does not work.